pkg/osvscanner/fixtures/filter/all/configs/a/osv-scanner.toml

```diff
@@ -0,0 +1,15 @@
+[[IgnoredVulns]]
+id = "GHSA-mc8h-8q98-g5hr"
+reason = "Ignore 1"
+
+[[IgnoredVulns]]
```

Comment:
nit: to make these tests easier to follow, can you please annotate these with comments to explain which ones are aliases of each other? 

Same comment for all other test osv-scanner.toml files.

Reply:
Added some annotations to all toml files